cromshell

```diff
@@ -313,6 +315,18 @@ function handleArgs_workflow_ids()
   fi
 }
 
+function matchRelativeWorkflowId() 
+{
+  local rv=1
+
+  if [[ "${1}" =~ ^-[[:xdigit:]]*$ ]] ; then
+    rv=0
+  fi
+
+  echo ${rv}
```

Comment:
why echo here?

Reply:
It keeps this function as a mirror of the `matchWorkflowId` function which also prints out the return value.  The echo can probably be removed from both.